cudaaligner/src/myers_gpu.cu

```diff
@@ -335,28 +336,33 @@ int32_t myers_compute_edit_distance(std::string const& target, std::string const
     if (get_size(query) == 0)
         return get_size(target);
 
+    const int32_t n_words = (get_size(query) + word_size - 1) / word_size;
+    matrix<int32_t> score_host;
+
     cudaStream_t stream;
     CGA_CU_CHECK_ERR(cudaStreamCreate(&stream));
+    {
+        DefaultDeviceAllocator allocator;
 
-    int32_t max_sequence_length = std::max(get_size(target), get_size(query));
-    device_buffer<char> sequences_d(2 * max_sequence_length);
-    device_buffer<int32_t> sequence_lengths_d(2);
+        int32_t max_sequence_length = std::max(get_size(target), get_size(query));
+        device_buffer<char> sequences_d(2 * max_sequence_length);
```

Comment:
any reason these `device_buffer` calls aren't using the allocator?

Reply:
Nope, I forgot about those. Thanks! I'll fix it.